.travis.yml

```diff
@@ -1,10 +1,9 @@
 language: ruby
-sudo: false
```

Comment:
`sudo: false` is not needed anymore. See also https://blog.travis-ci.com/2018-11-19-required-linux-infrastructure-migration